packages/documentation/docs/concepts/accounting/assets.md

```diff
@@ -0,0 +1,34 @@
+# Assets
```

Comment:
should we replace this: https://deploy-preview-1748--brilliant-pasca-3e80ec.netlify.app/docs/concepts/asset with what you wrote here?